server/channels/app/email/helper_test.go

```diff
@@ -170,7 +172,7 @@ func (th *TestHelper) CreateTeam() *model.Team {
 	}
 
 	var err error
-	if team, err = th.store.Team().Save(team); err != nil {
+	if team, err = th.store.Team().Save(ctx, team); err != nil {
```

Comment:
Is `th.Context` not yet populated at this point?

Reply:
`th` is a [different `TestHelper`](https://github.com/mattermost/mattermost/blob/master/server/channels/app/email/helper_test.go#L23) that doesn't have a context.